### DISCUSSION TITLE
Screenshots in README

### DISCUSSION THREAD

**Discussion by phgz** [projekt0n/github-nvim-theme#279] [General]
Hi,

I was wondering if, since 1.0.0, the colours were changed a lot, since when I compare what I actually have (leaving everything as default (except `hide_end_of_buffer` and `transparent`), using `github_light` theme) to the screenshots, it is a bit off (comments, arguments, class/objects). So is it supposed to look like the actual screenshot, or these need to be updated?

Here what is looks like for me using the same code template as in the README.

<img width="858" alt="Screenshot 2023-06-23 at 20 12 42" src="https://github.com/projekt0n/github-nvim-theme/assets/22773951/2b6cfeb6-c4e2-4382-bf91-b9ec3e60273a">

Thanks!

**Comment by ful1e5** [comment]
@phgz  Screenshots updated in 7e08e9cbf6da64b151f708a3e7e9f43447ae0171
